docs/01-start-here/06-faqs.md

```diff
@@ -0,0 +1,23 @@
+#FAQs
+
+## I can see mark-up in my response, but it's not in the application anywhere. Where is it coming from?
+
+Your markup is coming from CAPI and used as-is.
+
+For example: you're trying to style a class, but can't find it in the Frontend codebase and want to make sure it's only used in the HTML where you're looking.
+
+In this case, you can check in the `content-api` repo and your class will probably appear in some text fixtures and then you can look in the `flexible-content` repo (composer) where you'll probably see the view that adds the class you're styling.
+
+If your class only appears on the component you expected, you can be fairly confident that it won't appear anywhere else.
+
+## What are the make commands on Frontend?
+
+Look in the [makefile](https://github.com/guardian/frontend/blob/master/makefile)
+
+## How do I run JS, bundled and hashed, as I would see it on the live site?
+
+To see hashed JS bundles locally you can set assets.useHashedBundles=true in your frontend.conf and run make compile.
```

Comment:
Do you mean `frontend.properties`? frontend.conf is where I have my `devOverrides`...